spec/unit/resource/osx_profile_spec.rb

```diff
@@ -0,0 +1,60 @@
+#
+# Author:: Nate Walck (<nate.walck@gmail.com>)
+# Copyright:: Copyright (c) 2015 Facebook, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'spec_helper'
+
+describe Chef::Resource::OsxProfile do
+  before(:each) do
+    @resource = Chef::Resource::OsxProfile.new("Test Profile Resource")
```

Comment:
Same note here with `let` - rather than doing this in a `before` block, simple have `let(:resource) { Chef::Resource::OsxProfile.new("Test Profile Resource") }`